src/components/layer-controller/ChannelController.js

```diff
@@ -9,8 +9,8 @@ import ChannelOptions from './ChannelOptions';
 
 const COLORMAP_SLIDER_CHECKBOX_COLOR = [220, 220, 220];
 
-const toRgb = (on, arr) => {
-  const color = on ? COLORMAP_SLIDER_CHECKBOX_COLOR : arr;
+const toRgb = (on, theme, arr) => {
+  const color = (on || (theme === 'light' && arr.every(i => i === 255))) ? COLORMAP_SLIDER_CHECKBOX_COLOR : arr;
```

Comment:
Maybe add a comment on the conditions here? The function name doesn't really describe the functions effect (and didn't before). Perhaps we could rename the function to something like `toRgbUIString` ... although that is confusing too.

Reply:
Ditto. Also, should COLORMAP_SLIDER_CHECKBOX_COLOR be moved inside? I don't think it's used anywhere else.
```suggestion
  const color = (on || (theme === 'light' && arr.every(i => i === 255))) ? [220, 220, 220] : arr;
```

Reply:
(Slight preference for adding new parameters to the end of existing functions... If we've missed a usage, it will be less confusing if a latter argument just isn't filled in, than if the assignments in the middle have changed.

Reply:
(And should we distinguish between `themeName` and `theme`?)